main.go

```diff
@@ -63,11 +64,11 @@ func main() {
 		return
 	}
 	if err := vortex.LoadVariables(variablePath); err != nil {
-		fmt.Println("Unable to load files due to:", err)
+		color.Red("Unable to load %v variables", variablePath)
 		os.Exit(1)
 	}
 	if err := vortex.ProcessTemplates(templatePath, outputPath); err != nil {
-		fmt.Println("Unable to process templates due to:", err)
+		color.Red("Unable to process %v templates ", templatePath)
```

Comment:
Technical not sure because the error returned describes the reason why it failed, not file had issues.